controllers/user.js

```diff
@@ -1,20 +1,19 @@
 const User = require('../models/user');
-const Exercise = require('../models/workoutlog');
-//const Log = require('../models/workoutlog');
+const Exercise = require('../models/exercise');
+const Log = require('../models/log');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7841248)
```suggestion
const Log = require("../models/log");
```